WordPress/src/main/java/org/wordpress/android/GCMIntentService.java

```diff
@@ -232,14 +236,34 @@ protected void handleDefaultPush(Context context, Bundle extras) {
             mBuilder.setLights(0xff0000ff, 1000, 5000);
         }
 
+        mBuilder.setCategory(NotificationCompat.CATEGORY_SOCIAL);
+
         PendingIntent pendingIntent = PendingIntent.getActivity(context, 0, resultIntent,
                 PendingIntent.FLAG_CANCEL_CURRENT | PendingIntent.FLAG_UPDATE_CURRENT);
         mBuilder.setContentIntent(pendingIntent);
+
         NotificationManager mNotificationManager =
                 (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
         mNotificationManager.notify(PUSH_NOTIFICATION_ID, mBuilder.build());
     }
 
+    // Returns true if the note type is known to have a gravatar
+    public boolean shouldCircularizeNoteIcon(String noteType) {
+        if (TextUtils.isEmpty(noteType)) return false;
+
+        switch (noteType) {
```

Comment:
We really should have constants for these note type strings (esp. "c" since it's hard-coded a few times here).